src/fprime/fbuild/cli.py

```diff
@@ -50,6 +50,9 @@ def run_fbuild_cli(
         make_args: arguments to supply to the build tool (make or ninja)
     """
     if parsed.command == "generate":
+        if parsed.force:
+            purge(build, parsed)
```

Comment:
The `build` object has a `purge()` method that basically just calls `rm -rf` on that specific build. I wonder if it would be better to use that (maybe with an additional info log). Otherwise `fprime-util generate --force` will wipe all build caches, and not just the one we are trying to force-generate.

Reply:
Agreed.  Can we use that?